website/src/logged_in/prediction_tool/CreatePrediction.js

```diff
@@ -123,8 +130,17 @@ function CreatePrediction(props) {
                                                 fullWidth
                                                 onChange={(event) => setName(event.target.value)}
                                             />
+
+
                                         </Grid>
 
+                                        {Boolean(name != null) ||
```

Comment:
This is not the correct way to handle input validation for text fields. Material UI provides a mechanism for textfield validation, as you can see at [https://material-ui.com/components/text-fields/](https://material-ui.com/components/text-fields/).

Also, we need to check for not only if fields are empty, but if they contain invalid data. For instance, words should not be entered into forecasted attendance...that should only be numbers.